pkg/services/slack.go

```
@@ -196,7 +201,11 @@ func newSlackClient(opts SlackOptions) *slack.Client {
 	if opts.ApiURL != "" {
 		apiURL = opts.ApiURL
 	}
-	transport := httputil.NewTransport(apiURL, opts.InsecureSkipVerify)
+	var idleConnTimeout time.Duration
+	if opts.IdleConnTimeout != "" {
+		idleConnTimeout, _ = time.ParseDuration(opts.IdleConnTimeout)
```

Why are you suppressing the error?